### ISSUE TITLE
Large file support on 32-bits

### ISSUE THREAD

**Issue by vasi** [vasi/pixz#26]
Reported by email. I probably need something like "-D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE" on some platforms.

**Comment by wookietreiber** [comment]
@mrjones69 I just added a GNU autotools build process, which uses the autoconf macro `AC_SYS_LARGEFILE`. Would you please pull from current master, build the project (see README.md for instructions)? The macro should add these flags on 32-bit architectures automatically. Would you please verify this?

**Comment by mrjones69** [comment]
Confirmed using Fedora 22 i686 image.
The autoconf set __FILE_OFFSET_BITS to 64 in config.h.

Thanks!

**Comment by wookietreiber** [comment]
Have you also tried to compress / decompress such a large file?

**Comment by mrjones69** [comment]
Yes and sorry for the delayed response.
I kept messing up my VM before I could get to the actual test. :)

My test file is a 7GB .pxz archive that contains (among lesser files) 
one 23GB file.
This was created on another system using a 64-bit compile of the 
previously released version of pixz.

I was able to successfully decompress this archive on the 32-bit Fedora 
22 system using the latest pixz code.
Then using just the 23GB file as input, I compressed and decompressed it 
successfully using the 32-bit pixz.

Thank you for your efforts on this excellent little program!
